package.json

```diff
@@ -326,20 +331,26 @@
                         "default": 8000,
                         "description": "The port of a REST API app server to connect to"
                     },
-                    "marklogic.documentsDb": {
+                    "marklogic.unitTestPort": {
```

Comment:
For property name, let's just do "testPort" with a description of e.g. "Optional port for a REST API server that supports running marklogic-unit-test tests". This is based on ml-gradle using the names `mlRestPort` and `mlTestRestPort`, which a developer is probably making use of when using marklogic-unit-test. mlxprs doesn't use the word "rest" in the property names, which is fine, so I think "test port" would be the most consistent name to use between the two tools.